hal/lib/athena/Encoder.cpp

```diff
@@ -43,7 +43,11 @@ Encoder::Encoder(HAL_Handle digitalSourceHandleA,
                    digitalSourceHandleB, analogTriggerTypeB, reverseDirection,
                    encodingType, status);
 
-      m_encodingScale = encodingType == HAL_Encoder_k1X ? 1 : 2;
+      if (encodingType == HAL_Encoder_k1X) {
```

Comment:
Going to have to disagree with you on these: ternary is perfectly readable here in 1/5th the number of lines.